org-formation/090-systems-manager/QuickSetup-PatchPolicy.yaml

```diff
@@ -0,0 +1,95 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: An SSM patch policy to patch all instances in the given AWS Organization
+Parameters:
+  TargetOrgRoot:
+    Type: String
+    Description: >-
+      AWS Organization root ID to target.
+  TargetRegions:
+    Type: String
+    Description: >-
+      Comma separated list of AWS regions to target.
+Resources:
+  QuickSetupPatchLogBucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+  QuickSetupPatchLogBucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      Bucket: !Ref QuickSetupPatchLogBucket
```

Comment:
i assume this bucket will need to be excluded in security hub findings for `2.1.5.2 S3 Block Public Access setting`?

Reply:
no it shouldn't need an exception, blocking public access is set to true